backends/README.md

```diff
@@ -1,43 +1,43 @@
 # Metrics long term archiving
 
-Netdata supports backends for archiving the metrics, or providing long term dashboards, using Grafana or other tools,
-like this:
+Netdata supports long-term archiving and graphing of historical data, by introducing a feature called "Backends". **Note that the speed of the Backend does not not affect Netdata in any way.**
```

Comment:
We are changing our "backends" we are renaming to "Exporters", please, can you change this text?

Reply:
Please ignore the previous comment, let us finish the migration before.